vite.config.js

```diff
@@ -1,28 +1,15 @@
 import react from '@vitejs/plugin-react'
 import path from 'path'
-import { defineConfig, loadEnv } from 'vite'
+import { defineConfig } from 'vite'
 import svgr from 'vite-plugin-svgr'
 
 
 export default defineConfig(({ mode }) => {
-  const env = loadEnv(mode, process.cwd())
-
   return {
     build: {
       outDir: 'build'
     },
-    define: {
-      // Expose env as process.env instead of import.meta for jest
-      ...Object.entries(env).reduce(
-        (prev, [key, val]) => {
-          return {
-            ...prev,
-            [`process.env.${key}`]: `"${val}"`
-          }
-        },
-        {}
-      )
-    },
```

Comment:
This never worked, but didn't cause any errors because we didn't have anything that attempted to read environment variables in the app.

Any attempt to read `process.env.SOME_VARIABLE` would fail if no environment variables were defined because `process` would be undefined.